apps/portals/my-pages/src/components/Header/Header.tsx

```
@@ -9,48 +9,89 @@ import {
   const hasNotificationsDelegationAccess = user?.scopes?.includes(
     DocumentsScope.main,
   )
+  const [show, setShow] = useState<boolean>(true)
+
+  const [hide, setHide] = useState<boolean>(true)
+
+  useScrollPosition(
+    ({ prevPos, currPos }) => {
+      let px = -600
+
+      if (typeof window !== `undefined`) {
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal

(lint/suspicious/useValidTypeof)